TESTS/usb_device/msd/mbed_app.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Per-tests configs aren't allowed. You may be able to use the test configs at https://github.com/ARMmbed/mbed-os/tree/master/tools/test_configs. @bridadan or @theotherjimmy may have more details around this.

Reply:
Unfortunately that mechanism isn't really documented. It seems like you can add an entry to the "default_test_configuration" but I haven't used it personally.

Reply:
What about `--test-config <TEST_CONFIG>` ?

Reply:
I believe so yes. If you don't specify a `--test-config` or an `--app-config` it seems to go with the default one.